pkg/encoding/v1/encoding_test.go

```diff
@@ -397,6 +397,30 @@ volumes: []
 			"",
 			nil,
 		},
+		{
+			true, `
+version: 0.1-dev
+services:
+- name: helloworld
+  replicas: 0
+  containers:
+  - image: tomaskral/nonroot-nginx
+`,
+			&object.OpenCompose{
+				Version: Version,
+				Services: []object.Service{
+					{
+						Name: "helloworld",
+						Containers: []object.Container{
+							{
+								Image: "tomaskral/nonroot-nginx",
+							},
+						},
+						Replicas: 0,
```

Comment:
since using values as it is so not much changes, checks done at a level up when creating `deployment`, so original info is not lost from opencompose as well as in cluster.